zucchini-ui-backend/src/main/java/io/zucchiniui/backend/scenario/domain/Scenario.java

```diff
@@ -7,14 +7,7 @@
 import org.mongodb.morphia.annotations.Id;
 
 import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Use real import, not wildcard imports.

Configure Intellij like this :

![image](https://user-images.githubusercontent.com/800987/28778297-ff0ed5d2-75fe-11e7-9962-6ddbe39eb686.png)